client/src/components/geneExpression/index.tsx

```diff
@@ -175,7 +176,14 @@ class GeneExpression extends React.Component<{}, State> {
     const { isCellGuideCxg } = this.props;
     const { geneSetsExpanded, markerGeneSetsExpanded } = this.state;
     return (
-      <div>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          justifyContent: "space-between",
+          padding: globals.leftSidebarSectionPadding,
```

Comment:
we're using a lsb global on the rsb, should we be using a different one or normailize this global?